internal/errors/errors.go

```diff
@@ -510,6 +510,9 @@ var (
 	FFCRequestTypeNotImplemented = e(100217, "FFCAPI request '%s' not currently supported")
 	// FFCBlockNotAvailable is returned when a receipt is not found
 	FFCBlockNotAvailable = e(100218, "Block not available")
+
+	// ReceiptStoreLevelDBKeyNotUnique idepotency check failed on request ID
```

Comment:
```suggestion
	// ReceiptStoreLevelDBKeyNotUnique idempotency check failed on request ID
```
..in the spirit of the other change in this PR ;)

Reply:
Guess I can only correct that spelling once :)